tests/rkt_pid_file_test.go

```diff
@@ -103,6 +106,24 @@ func TestPidFileDelayedStart(t *testing.T) {
 
 // Check that "enter" doesn't wait forever for the ppid file when the pod is terminated
 func TestPidFileAbortedStart(t *testing.T) {
+
+	var (
+		escapeSequence string
+		processFail    bool
+	)
+
+	// Configure escape sequence and expected exit code for flavors
+	if isKVM() {
+		// For kvm escape character is ^Ax. This process will exit with code 0
+		// If pod won't stop after sending escape sequence, than test will wait
+		// till go test timeout.
+		escapeSequence = "\001\170"
+		processFail = false
```

Comment:
s/processFail/shouldSucceed and explicit information in docs that this behaviors is different from original nspawn